source/features/collapse-markdown-sections.css

```diff
@@ -0,0 +1,7 @@
+.markdown-body > :is(h1, h2, h3, h4, h5, h6) {
+	cursor: pointer;
+}
+
+.markdown-body :is(a, details) :is(h1, h2, h3, h4, h5, h6) {
```

Comment:
> Blue box has correct cursor red does not. Is that correct?

No. We should not change that cursor since it's still natively clickable, it's just not managed by us

I think we can just drop `details` from here since they always show `pointer` anyway, even if we don't handle it
```suggestion
.markdown-body a :is(h1, h2, h3, h4, h5, h6) {
```